Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/Microsoft_Defender_For_Endpoint_-_Collect_investigation_package_.yml

```diff
@@ -0,0 +1,440 @@
+      {
+        "position": {
+          "x": 162.5,
+          "y": 90
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 1764db60-72b7-4693-8736-b56f19c2ed75
+    type: condition
+    task:
+      id: 1764db60-72b7-4693-8736-b56f19c2ed75
+      version: -1
+      name: Auto collect investigation package from the Endpoint?
+      type: condition
+      description: Validate if the auto-collect is set
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "3"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: inputs.AutoCollectinvestigationPackege
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    view: |-
+      {
+        "position": {
+          "x": 390,
+          "y": 555
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 38958f91-d5d1-419f-893d-010071d05491
+    type: regular
+    task:
+      id: 38958f91-d5d1-419f-893d-010071d05491
+      version: -1
+      name: Collect investigation Package
+      description: Collect an investigation package from a machine.
+      script: '|||microsoft-atp-collect-investigation-package'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      comment:
+        simple: 'XSOAR - Collect Investigation Package - inc #${incident.ID}'
+      machine_id:
+        complex:
+          root: Endpoint
+          accessor: ID
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 390,
+          "y": 905
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 4db57d68-6de7-4fa9-87a3-377329afdb37
+    type: condition
+    task:
+      id: 4db57d68-6de7-4fa9-87a3-377329afdb37
+      version: -1
+      name: Approve Investigation package collection
+      description: Approve Investigation package collection
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "8"
+      "Yes":
+      - "3"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 610,
+          "y": 730
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+        simple: Approve isolation
```

Comment:
related?

Reply:
Wrong display message (good catch). Changing this now.